CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contributing
```

Comment:
I have added this file by following this GitHub guide: https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors#adding-a-contributing-file

Reply:
Thanks. That’s great.